tika/server.go

```diff
@@ -127,18 +127,13 @@ var command = exec.Command
 // cancelled.
 func (s *Server) Start(ctx context.Context) error {
 
-	var props = ""
-
-	//Check to see if there are any tuples in the servers java props, if so
-	//format and add to the props variable
-
-	if len(s.Javaprops) > 0 {
-		for i := range s.Javaprops {
-			props = fmt.Sprintf("-D%s=%s ", i, s.Javaprops[i])
-		}
+	props := []string{}
+	for k, v := range s.JavaProps {
+		props = append(props, fmt.Sprintf("-D%s=%s", k, v))
 	}
+	propsArgs := strings.Join(props, " ")
 
-	cmd := command("java", append([]string{props, "-jar", s.jar, "-p", s.port}, s.child.args()...)...)
+	cmd := command("java", append([]string{propsArgs, "-jar", s.jar, "-p", s.port}, s.child.args()...)...)
```

Comment:
I think all of the props should be passed as separate args? Probably need another `append`:

```suggestion
	cmd := command("java", append(append(props, []string{propsArgs, "-jar", s.jar, "-p", s.port}, s.child.args()...)...)...)
```